src/store/builder/index.ts

```diff
@@ -309,6 +304,14 @@ export const actions: ActionTree<IBuilderState, IRootState> = {
 
     return newFlowContainer ?? undefined
   },
+
+  setBlockPositionTo({commit, rootGetters}, {position: {x, y}, block}) {
+    commit('flow/flow_updateBlockCoordinates', {
+      flowId: rootGetters['flow/activeFlow'].uuid,
+      blockId: block.uuid,
+      coordinates: {x, y},
+    }, {root: true})
+  },
```

Comment:
Making this an action allows committing a mutation in the neighbor `flow` module.